Update japan.md (#9941)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/support/japan.md

## Phone number
0120 996 680 (Office 365 Enterprise)

> 0120 628 860 (Other Plans) (Example : Microsoft 365 Business Basic, Standard, Premium etc)

Reviewer: ```suggestion
0120 628 860 (Other plans, such as Microsoft 365 Business Basic, Standard, Premium, and others)
```

English: 24 hours a day, 7 days a week

> **Other Plans:** (Example : Microsoft 365 Business Basic, Standard, Premium etc):

Reviewer: ```suggestion
**Other plans** (such as Microsoft 365 Business Basic, Standard, Premium, and others):
```
